### PR TITLE
OCPBUGS#13668: Added note on SDN and plain OVN are not supported

### PR DESCRIPTION
**Pull request by mburke5678**
https://issues.redhat.com/browse/OCPBUGS-13668?

Removed sentence that SDN is the default.  Added note that OVN with hydrid and SDN are not supported. 

Current: WMCO release notes -> [Supported networking](https://docs.openshift.com/container-platform/4.12/windows_containers/windows-containers-release-notes-7-x.html#supported-networking)
Preview: WMCO release notes -> [Supported networking](http://file.rdu.redhat.com/mburke/wmco-clarify-networking-412/windows_containers/windows-containers-release-notes-7-x.html#supported-networking)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Review by jfrancoa — APPROVED** [review]
lgtm

**Comment by lahinson** [comment]
Hi @mburke5678 - Do you have a preview link for this one?

File: modules/wmco-prerequisites.adoc [hunk 1]
```diff
@@ -37,8 +37,12 @@ a|* Windows Server 2022, OS Build link:https://support.microsoft.com/en-us/topic
 
 == Supported networking
 
-Hybrid networking with OVN-Kubernetes is the only supported networking configuration. See the additional resources below for more information on this functionality. The following tables outline the type of networking configuration and Windows Server versions to use based on your platform. You must specify the network configuration when you install the cluster. Be aware that OpenShift SDN networking is the default network for {product-title} clusters. However, OpenShift SDN is not supported by WMCO.
+Hybrid networking with OVN-Kubernetes is the only supported networking configuration. See the additional resources below for more information on this functionality. The following tables outline the type of networking configuration and Windows Server versions to use based on your platform. You must specify the network configuration when you install the cluster. 
 
+[NOTE]
+====
+The WMCO does not support OVN-Kubernetes without hybrid networking or OpenShift SDN.
```
Comment (lahinson) [inline modules/wmco-prerequisites.adoc:44]:
```suggestion
WMCO does not support OVN-Kubernetes without hybrid networking or OpenShift SDN.
```

As far as I see in the related docs, "WMCO" doesn't take an article. Also, should "OpenShift" be replaced by a product name variable here?

**Review by lahinson — COMMENTED** [review]
A couple of minor suggestions, but looks pretty good overall.

File: modules/wmco-prerequisites.adoc (hunk 1 above)
Comment (mburke5678) [reply modules/wmco-prerequisites.adoc:44]: OpenShift SDN appears to be correct, based on an Atom search. There are only 5 instances of {product-title} SDN, compared to 191 instances of OpenShift SDN.
Also, there are scores of instances of _the WMCO_. I think it reads better with the article. WDYT?

Comment (lahinson) [reply modules/wmco-prerequisites.adoc:44]: Thanks for checking on OpenShift SDN and WMCO. I'm fine with keeping "OpenShift SDN" if that's what we usually use.

I scanned over the page again, and you're right -- there are plenty of instances of "the WMCO". I did spot a few instances where an article wasn't included, though, which is why I initially thought it didn't take one. I checked the Red Hat naming guidelines, and it includes the article before WMCO, so feel free to disregard my suggestion to remove the article.

**Pull request merged**